CLAUDE.md

```
@@ -0,0 +1,149 @@
+podman run -d --name argocd-mcp-server \
+  -e ARGOCD_MCP_SERVER_LISTEN_HOST=0.0.0.0 \
+  -e ARGOCD_URL=<url> \
+  -e ARGOCD_TOKEN=<token> \
+  -e ARGOCD_MCP_DEBUG=true \
+  -p 8080:8080 argocd-mcp-server:latest
+```
+
+## Code Architecture
+
+### Package Structure
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)